controllers/campgrounds.js

```diff
@@ -93,7 +94,7 @@ module.exports.showCampground = async (req, res) => {
         req.flash('error','Cannot find that campground!');
         return res.redirect('/campgrounds');
     }
-    res.render('campgrounds/show',{campground});
+    res.render('campgrounds/show',{campground,userRole,adminRole});
```

Comment:
This change is not required

Reply:
But the show.ejs file is giving error and not rendering the page saying userRole , adminRole not defined.
Hence I passed those variables and it is getting rendered without giving any error. 
Should I remove it? 
![Screenshot (2721)](https://github.com/user-attachments/assets/f2f9934e-bf05-49e1-87a8-e0df7ccc8981)
Kindly review the error and guide me if I should remove those passed variables?